cluster/addons/fluentd-elasticsearch/fluentd-es-image/Makefile

```diff
@@ -4,7 +4,7 @@ IMAGE = fluentd-elasticsearch
 TAG = 1.11
```

Comment:
@thockin Any rhyme or reason to the versioning here, or would it make sense for me to bump this as well?  If the intent is for this to reflect a version of ES or fluentd, I don't want to mess with it.